packages/realm-server/billing/stripe-webhook-handlers/subscribe.ts

```diff
@@ -0,0 +1,20 @@
+import { DBAdapter } from '@cardstack/runtime-common';
+import {
+  insertStripeEvent,
+  updateUserStripeCustomerId,
+} from '../billing_queries';
+import { StripeEvent } from './';
+
+export async function handleCheckoutSessionCompleted(
+  dbAdapter: DBAdapter,
+  event: StripeEvent,
+): Promise<Response> {
+  await insertStripeEvent(dbAdapter, event);
```

Comment:
In https://github.com/cardstack/boxel/pull/1719 I introduce an `stripe_events.is_processed`field so that we can ensure idempotency for handlers.

In there I also introduced a transaction manager to make sure that if something fails in the middle of the handler we don't get to a messed up state in the DB.

To avoid conflicts I think it's ok if you add a ticket to address these 2 issues after my PR is merged (I suppose this one will be merged before mine)